content/docs/0.5.0/reference/cli/index.md

```diff
@@ -261,15 +261,24 @@ keptn configure monitoring prometheus --project=PROJECTNAME --service=SERVICENAM
 
 ### keptn send event new-artifact
 
-After onboarding a service, the Keptn CLI allows pushing a new artifact for the service. This artifact is a Docker image, which can be located at Docker Hub, Quay, or any other registry storing docker images. The new artifact is pushed in the first stage specified in the `shipyard.yaml` file (usually this will be the dev stage). Afterwards, Keptn takes care of deploying this new artifact to the other stages.
+After onboarding a service, the Keptn CLI allows to trigger the deployment of an artifact.
+This artifact is a Docker image, which can be located at Docker Hub, Quay, or any other registry storing docker images. The new artifact is pushed in the first stage specified in the `shipyard.yaml` file (usually this will be the dev stage). Afterwards, Keptn takes care of deploying this new artifact to the other stages.
```

Comment:
> or any other registry storing docker images

Can we change this to "or basically any container registry which is accessible from within the cluster."

Reason: "any other registry" does not exclude your local docker registry on your computer.

Reply:
Thx